content/docs/user-guide/settings/az-console-how-to-use-settings-registry.md

```diff
@@ -0,0 +1,173 @@
+    { "op": "add", "path": "/O3DE/Autoexec/ConsoleCommands/LoadLevel", "value": "levels/levelname/levelname.spawnable" },
+    { "op": "add", "path": "/O3DE/Autoexec/ConsoleCommands/bg_ConnectToAssetProcessor", "value": true }
+]
+```
+
+Ex. Executing Console Commands from a config or Settings Registry file
+----------------------------------------------------------------------
+
+Config files as well Settings Registry files with the preceding content be executed using a single ExecuteConfigFile function in the AZ Console
+
+**Executing Console Commands**
+
+
+```c++
+auto console = AZ::Interface<AZ::IConsole>::Get();
+console.ExecuteConfigFile(<path to [user.cfg|user.setreg|user.setregpatch]>);
+```
+
+Order in which ConsoleCommands are executed from a single file
+
+Console commands within a specific config or settings registry file are executed in the order they appear in those files from top to bottom.
+
+Here is an example of executing three console commands using the JSON Merge Patch syntax
+
+**Ordering snippet**
+
+```json
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "testInit": 3,
+                "testBool": true,
+                "testChar": "Q"
+            }
+        }
+    }
+}
+```
+
+Those commands would execute in order that were seen in the JSON file(testInit → testBool → testChar)
+
+Running Console Commands via updating a value within the Settings Registry
+==========================================================================
+
+Running console commands can also be done via modifying keys within the Settings Registry, that are under the "/O3DE/Autoexec/ConsoleCommands" JSON object
+
+Ex. Using the Settings Registry to modify the "t\_scale" CVar
+-------------------------------------------------------------
+
+**CVar Modification**
+
+```c++
+//! The AZ::IConsole::ConsoleAutoexecCommandKey variable is set to the Settings Registry Console commands root key:
+//! "/O3DE/Autoexec/ConsoleCommands"
+using FixedValueString = AZ::SettingsRegistryInterface::FixedValueString;
+constexpr auto tScaleCVar = FixedValueString(AZ::IConsole::ConsoleRootCommandKey) + "/t_scale;
+auto SettingsRegistry = AZ::SettingsRegistry::Get();
+settingsRegistry->Set(tScaleCVar, "0.5");
+```
+
+Timing of when Console Commands are invoked
+===========================================
+
+The ComponentApplication loads .setreg(patch) files before any Gem Modules are loaded or activated in [ComponentApplication::Create](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L628-L636).
+The ComponentApplication also supports executing the "user.cfg" in the asset cache after Gem Modules are loaded, but before they activate. This can be seen in [ComponentApplication::CreateCommon](https://github.com/o3de/o3de/blob/stabilization/2110/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L692-L702).
```

Comment:
nit: avoid code links to stabilization branch, should point to last known release or dev. Be consistent with code links (ie pick last release or dev for all links(

Reply:
I'll update this to point to permalink in the development branch